lib/_stream_readable.js

```diff
@@ -458,6 +458,12 @@ Readable.prototype.read = function(n) {
       endReadable(this);
   }
 
+  // Drop data if '_dumped' flag exists. This is for http.IncomingMessage.
+  // When http.ClientRequest.abort() is called, the '_dumped' flag of
+  // http.IncomingMessage is set to 'true'.  Then http.ClientRequest
+  // shouldn't receive any 'data' event.
+  if (this._dumped === true) ret = null;
```

Comment:
`_dumped` is written from here: [lib/_http_incoming.js](https://github.com/nodejs/node/blob/master/lib/_http_incoming.js#L316) and it's also been used here: [lib/_http_common.js](https://github.com/nodejs/node/blob/master/lib/_http_common.js#L138). I am also curious about when this flag: `_dumped` is introduced. Could this flag be replaced by any of the flag in `ReadableState`? or move this flag into the `ReadableState`?